cli/command/run/cmd.go

```diff
@@ -849,3 +852,19 @@ func awaitGlobalTimeout() {
 		os.Exit(1)
 	}
 }
+
+// checkForUpdates check if there is a saucectl update available.
+func checkForUpdates() {
+	gh := github.Client{
+		HTTPClient: &http.Client{Timeout: githubTimeout},
+		URL:        "https://api.github.com",
+	}
+
+	v, err := gh.HasUpdateAvailable()
+	if err != nil {
+		return
+	}
+	if v != "" {
+		log.Warn().Msgf("An update of saucectl is available (%s)", v)
```

Comment:
Wording might be better this way:
```suggestion
		log.Warn().Msgf("An new version of saucectl is available (%s)", v)
```

Reply:
Updated to `A new version of saucectl is available (%s)`